scripts/Units/armguard.bos

```diff
@@ -92,36 +73,42 @@ aimCommon(heading, pitch)
 
 AimPrimary(heading, pitch)
 {
-	if( trajectoryMode != LOW_TRAJECTORY )
-	{
+	if (AimingState != AIMING_PREFERRED){
 		return(0);
 	}
 
 	signal SIGNAL_AIM;
 	set-signal-mask SIGNAL_AIM;
-	call-script aimCommon(heading, pitch);
 
-	// Prevent low-trajectory "bonus shots" while high-trajectory is reloading
+	call-script aimCommon(heading, pitch);
+	// Prevent "bonus shots" while alternative weapon is reloading
 	var reloading;
 	reloading = (get GAME_FRAME) < (get WEAPON_RELOADSTATE(2));
 	if( reloading )
 	{
```

Comment:
Not in the scope of this PR, just a general comment, but linked reload sounds like a useful general gadget to have.

Reply:
What is a linked reload?

Reply:
Preventing "bonus shots" while alternative weapon is reloading